pytest_django/compat.py

```diff
@@ -1,8 +1,29 @@
 # This file cannot be imported from until Django sets up
-try:
-    # Django 1.11
-    from django.test.utils import setup_databases, teardown_databases  # noqa: F401
-except ImportError:
+
+
+def _get_setup_and_teardown_databases():
```

Comment:
Refactor code such that we can use `return` to avoid deeply nested `try-except` blocks.